content/influxdb/v2.0/tools/chronograf.md

```diff
@@ -30,6 +30,9 @@ For example, you **cannot** do the following:
 - Modify retention policies
 - Add users
 
+When connected to an InfluxDB Cloud or InfluxDB 2.0 database, the **InfluxDB Admin**
+section of Chronograf is disabled.
+
 To complete administrative tasks, use the following:
 
 - **InfluxDB user interface (UI)**
```

Comment:
Link to the CLI in the v2 docs (as opposed to GitHub), Link to the API docs (rather than github)...